src/pages/join.jsx

```diff
@@ -0,0 +1,187 @@
+      description:
+        'Ensure your work continues and transfer your tasks to another!',
+      link: 'Join Slack',
+    },
+  ]
+
+  const icons2 = [
+    {
+      img: active,
+      title: 'Active Projects',
+      description:
+        'Check out the projects that are actively recruiting volunteers.',
+    },
+    {
+      img: codeconduct2,
+      title: 'Code of Conduct',
+      description: 'Review and agree to our code of conduct.',
+    },
+    {
+      img: slack,
+      title: 'Join Our Slack',
+      description:
+        'Check out our Slack where all of our collaboration happens.',
+    },
+  ]
+  return (
+    <Layout>
+      <Hero
+        eyebrowText="JOIN US"
+        mainHeading="Be part of the change in your commmunity"
+        // subHeading="Start the process of joining one of our on-going projects."
+        buttonText="Start Onboarding"
+      />
+      <Heading
+        browText="Volunteer Expectations"
+        headingText="Our culture is scrappy"
+        // description="We try our best to be organized, but we rely on our volunteers to shape how they want to contribute."
+        description="This is content that describes this section."
+        headingLevel="3"
+      />
+      <IconGrid icons={icons} />
+      {/* <IconTile
+          icon={code}
+          title={'Code of Conduct'}
+          description={'We ensure we keep teamwork safe and fun.'}
+        />
+        <IconTile
+          icon={question}
+          title={'Ask Questions'}
+          description={'Don’t be afraid to ask someone questions! '}
+        />
+        <IconTile
+          icon={available}
+          title={'Availability'}
+          description={
+            'We know life happens so let your team know if you have capacity.'
+          }
+        />
+        <IconTile
+          icon={handOff}
+          title={'Hand-Off Tasks'}
+          description={
+            'Ensure your work continues and transfer your tasks to another!'
+          }
+        /> */}
+      <ContentSection
+        eyebrowText={'PART 1: START ONBOARDING REGISTRATION'}
+        mainHeading={'To start onboarding, complete our onboarding form'}
+        content={
+          'The form gathers basic info like which project you want to join and your availability. Someone will review your response within 7 actual days and follow-up in Slack.'
+        }
+        linkUrl={'/components'}
+        linkText={'Start Onboarding'}
+        imgUrl={form}
+        imageDescription={'picture of the onboarding form step 1'}
+      />
+      <ContentSection
+        eyebrowText={'PART 2: VOLUNTEER SKILLS'}
+        mainHeading={"Next, we'll learn more about your specific skills"}
+        content={
+          'Someone will reach out in Slack to assign you a new task of completing another form. This will give us a better idea of which project to place you.'
+        }
+        linkUrl={'/components'}
+        linkText={'Join Our Slack'}
+        imgLeft={true}
```

Comment:
No need to add the `={true}`. If there is just an `imgLeft` it will resolve as true. And then if there isn't an `imgLeft` it will obviously resolve to false in the child component.